fossunited/api/hackathon.py

```diff
@@ -342,3 +342,62 @@ def join_team_via_code(team_code: str, user: str):
 
     team.append("members", {"member": participant.name})
     team.save()
+
+
+@frappe.whitelist()
+def get_session_participant(hackathon: str) -> dict:
+    """
+    Get participant details of the current session user
+
+    Args:
+        hackathon (str): Hackathon ID
+
+    Returns:
+        dict: Participant document as a dictionary
+    """
+    participant = frappe.db.get_value(
+        "FOSS Hackathon Participant",
+        {"hackathon": hackathon, "user": frappe.session.user},
+        [
+            "name",
+            "user_profile",
+            "full_name",
+            "email",
+            "is_student",
+            "git_profile",
+            "organization",
+            "hackathon",
+            "wants_to_attend_locally",
+            "localhost",
+            "localhost_request_status",
+        ],
+        as_dict=1,
+    )
+    return participant
+
+
+@frappe.whitelist()
+def delete_project(hackathon: str, team: str):
```

Comment:
I would unittest this in the following way - test that only the owner of a project can delete it and that a user who doesn't own the project cannot delete it.

Reply:
If this is a case @harshtandiya - then an add-on to an owner leaving a team would be, someone else be made an owner ? 

A project is an asset of a team, and anyone in the team can create a project right ?

Reply:
let's not set `owner` field unless required. Had a word with Hussain, he suggested to not mingle with owner fields. 

For now, i think anyone on the team should be able to trash the project if so required.